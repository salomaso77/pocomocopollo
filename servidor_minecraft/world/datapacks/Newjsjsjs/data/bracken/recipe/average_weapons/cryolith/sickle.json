{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:polar_bear_spawn_egg",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:max_damage": 200,
			"minecraft:item_name": "Cryolith Sickle",
			"minecraft:enchantments": {
				"minecraft:vanishing_curse": 1,
				"bracken:freezing_fix": 1,
				"bracken:fragility_curse": 1
			},
			"minecraft:item_model": "bracken:average_weapons/cryolith/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "12 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:cryolith_sickle",
					"amount": 11,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:cryolith_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}