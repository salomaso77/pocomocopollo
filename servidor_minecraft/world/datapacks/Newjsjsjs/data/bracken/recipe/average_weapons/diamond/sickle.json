{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" ##",
		"/  "
	],
	"key": {
		"#": "minecraft:diamond",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:diamond_sword",
		"components": {
			"minecraft:item_name": "Diamond Sickle",
			"minecraft:item_model": "bracken:average_weapons/diamond/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "9 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:diamond_sickle",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:diamond_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}