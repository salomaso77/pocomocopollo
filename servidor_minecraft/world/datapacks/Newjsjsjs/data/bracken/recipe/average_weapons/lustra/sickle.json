{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:sheep_spawn_egg",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:max_damage": 3000,
			"minecraft:enchantment_glint_override": true,
			"minecraft:item_name": "Lustra Sickle",
			"minecraft:item_model": "bracken:average_weapons/lustra/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "6 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:lustra_sickle",
					"amount": 5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:lustra_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}