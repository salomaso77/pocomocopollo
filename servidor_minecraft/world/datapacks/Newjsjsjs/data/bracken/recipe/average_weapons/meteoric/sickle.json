{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" ##",
		"/  "
	],
	"key": {
		"#": "minecraft:netherite_scrap",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Meteoric Sickle",
			"minecraft:item_model": "bracken:average_weapons/meteoric/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:meteoric_sickle",
					"amount": 9,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:meteoric_sickle",
					"amount": -3,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}