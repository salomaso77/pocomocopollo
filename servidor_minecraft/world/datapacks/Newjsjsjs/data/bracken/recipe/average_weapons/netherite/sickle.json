{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_ingot",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Netherite Sickle",
			"minecraft:item_model": "bracken:average_weapons/netherite/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_sickle",
					"amount": 9,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}