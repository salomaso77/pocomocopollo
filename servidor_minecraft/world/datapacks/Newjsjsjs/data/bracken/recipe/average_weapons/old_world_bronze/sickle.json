{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:host_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:cat_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Old World Bronze Sickle",
			"minecraft:item_model": "bracken:average_weapons/old_world_bronze/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_sickle",
					"amount": 9.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
				
		}
	}
}