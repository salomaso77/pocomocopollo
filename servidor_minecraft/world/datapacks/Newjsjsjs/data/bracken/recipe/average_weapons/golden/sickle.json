{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" ##",
		"/  "
	],
	"key": {
		"#": "minecraft:gold_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Sickle",
			"minecraft:item_model": "bracken:average_weapons/golden/sickle",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "6 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_sickle",
					"amount": 5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_sickle",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}